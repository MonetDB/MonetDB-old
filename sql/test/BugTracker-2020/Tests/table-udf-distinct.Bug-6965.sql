START TRANSACTION;
CREATE OR REPLACE FUNCTION test(i integer) 
RETURNS TABLE (value integer)
BEGIN
  return values (1), (2), (2), (3), (i);
END;

select distinct value from test(3);
ROLLBACK;
