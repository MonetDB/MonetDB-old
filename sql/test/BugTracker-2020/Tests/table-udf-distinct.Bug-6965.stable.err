stderr of test 'table-udf-distinct.Bug-6965` in directory 'sql/test/BugTracker-2020` itself:


# 12:23:31 >  
# 12:23:31 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-261797" "--port=34289"
# 12:23:31 >  


# 12:23:31 >  
# 12:23:31 >  "Done."
# 12:23:31 >  

