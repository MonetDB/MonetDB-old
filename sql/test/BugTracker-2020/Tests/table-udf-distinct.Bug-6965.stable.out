stdout of test 'table-udf-distinct.Bug-6965` in directory 'sql/test/BugTracker-2020` itself:


# 12:23:31 >  
# 12:23:31 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-261797" "--port=34289"
# 12:23:31 >  

#START TRANSACTION;
#CREATE OR REPLACE FUNCTION test(i integer) 
#RETURNS TABLE (value integer)
#BEGIN
#  return values (1), (2), (2), (3), (i);
#END;
#select distinct value from test(3);
% .%2 # table_name
% value # name
% int # type
% 1 # length
[ 1	]
[ 2	]
[ 3	]
#ROLLBACK;

# 12:23:31 >  
# 12:23:31 >  "Done."
# 12:23:31 >  

